src/booleans.js

```diff
@@ -1,61 +1,65 @@
 const negate = (a) => {
-  // your code here
+  return !(a);
 };
 
 const both = (a, b) => {
-  // your code here
+  return (a && b);
 };
 
 const either = (a, b) => {
-  // your code here
+  return (a || b);
 };
 
 const none = (a, b) => {
-  // your code here
+  return !(a || b);
 };
 
 const one = (a, b) => {
-  // your code here
+  return (a || b) && (!a || !b);
 };
 
 const truthiness = (a) => {
-  // your code here
+  if(!a)
+    { return false}
+  else 
+  { return true;}
```

Comment:
In any scenario where a function will either return true or false, there's usually an easier way. This could be done with simply:

```js
const truthiness = (a) => {
  return !!a;
}
```